scripts/before_pectra_upgrade_holesky.py

```diff
@@ -0,0 +1,510 @@
+        ),
+        (
+            "13. Change Vote time from 900 to 1080 on Aragon Voting 0xdA7d2573Df555002503F29aA4003e398d28cc00f",
+            (
+                contracts.voting.address,
+                contracts.voting.unsafelyChangeVoteTime.encode_input(NEW_VOTE_DURATION),
+            ),
+        ),
+        (
+            "14. Change Objection Phase time from 300 to 360 on Aragon Voting 0xdA7d2573Df555002503F29aA4003e398d28cc00f",
+            (
+                contracts.voting.address,
+                contracts.voting.unsafelyChangeObjectionPhaseTime.encode_input(NEW_OBJECTION_PHASE_DURATION),
+            ),
+        ),
+        (
+            "15. Revoke UNSAFELY_MODIFY_VOTE_TIME_ROLE from Aragon Voting (proxy) 0xdA7d2573Df555002503F29aA4003e398d28cc00f",
+            encode_permission_revoke(
+                target_app=contracts.voting,
+                permission_name="UNSAFELY_MODIFY_VOTE_TIME_ROLE",
+                revoke_from=contracts.voting,
+            ),
+        ),
+        (
+            "16. Grant CONFIG_MANAGER_ROLE on OracleDaemonConfig 0xC01fC1F2787687Bc656EAc0356ba9Db6e6b7afb7 to Aragon Agent (proxy) 0xE92329EC7ddB11D25e25b3c21eeBf11f15eB325d",
+            agent_forward(
+                [
+                    encode_oz_grant_role(
+                        contract=contracts.oracle_daemon_config,
+                        role_name="CONFIG_MANAGER_ROLE",
+                        grant_to=contracts.agent,
+                    )
+                ]
+            ),
+        ),
+        (
+            "17. Update the FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT parameter in the OracleDaemonConfig contract 0xC01fC1F2787687Bc656EAc0356ba9Db6e6b7afb7 to 0x08CA (2250)",
+            agent_forward(
+                [
+                    (
+                        contracts.oracle_daemon_config.address,
+                        contracts.oracle_daemon_config.update.encode_input(
+                            "FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT",
+                            convert.to_bytes(FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT_NEW_VALUE),
+                        ),
+                    )
+                ]
+            ),
+        ),
+        (
+            "18. Revoke CONFIG_MANAGER_ROLE on OracleDaemonConfig 0xC01fC1F2787687Bc656EAc0356ba9Db6e6b7afb7 from Aragon Agent (proxy) 0xE92329EC7ddB11D25e25b3c21eeBf11f15eB325d",
+            agent_forward(
+                [
+                    encode_oz_revoke_role(
+                        contract=contracts.oracle_daemon_config,
+                        role_name="CONFIG_MANAGER_ROLE",
+                        revoke_from=contracts.agent,
+                    )
+                ]
+            ),
+        ),
+        # Change GateSeal on WithdrawalQueue and ValidatorsExitBusOracle
+        (
+            "19. Grant PAUSE_ROLE on WithdrawalQueue 0xc7cc160b58F8Bb0baC94b80847E2CF2800565C50 to the new GateSeal 0xAE6eCd77DCC656c5533c4209454Fd56fB46e1778",
```

Comment:
Descriptions in items 19–22 contain `&nbsp;` instead of regular spaces. Is this intentional?